compiler/src/main/java/dagger/internal/codegen/BindingGraph.java

```diff
@@ -208,17 +208,17 @@ private BindingGraph create(
           : ImmutableSet.<TypeElement>of();
       for (TypeElement componentDependency : componentDependencyTypes) {
         explicitBindingsBuilder.add(provisionBindingFactory.forComponent(componentDependency));
-        List<ExecutableElement> dependencyMethods =
-            ElementFilter.methodsIn(elements.getAllMembers(componentDependency));
-        for (ExecutableElement method : dependencyMethods) {
-          // MembersInjection methods aren't "provided" explicitly, so ignore them.
-          if (isComponentContributionMethod(elements, method)) {
-            explicitBindingsBuilder.add(
-                componentDescriptor.kind().equals(PRODUCTION_COMPONENT)
-                        && isComponentProductionMethod(elements, method)
-                    ? productionBindingFactory.forComponentMethod(method)
-                    : provisionBindingFactory.forComponentMethod(method));
-          }
+      }
+      ImmutableSet<ExecutableElement> dependenciesMethods
```

Comment:
I assumed you meant you wanted me to perform the following renames:
* dependenciesMethods --> dependencyMethods
* method --> dependencyMethod.